src/turbolinks/index.coffee

```diff
@@ -7,7 +7,9 @@
 
 @Turbolinks =
   supported: do ->
-    window.history.pushState? and window.requestAnimationFrame?
+    window.history.pushState? and
+      window.requestAnimationFrame? and
+      window.addEventListener?
```

Comment:
Not technically needed to fix the issue, but seems reasonable to add for clarity.